.github/CONTRIBUTING.md

```diff
@@ -106,51 +76,50 @@ The`MONGODB_URL` will be `mongodb://USERNAME:PASSWORD@localhost:27017/DATABASE_N
 
 You can access mongo-express by visiting [localhost:8081](http://localhost:8081/).
 
-### Privileged Gateway Intents
```

Comment:
Privileged Gateway Intents still needs to exist

Reply:
This was my bad when shifting things around, forgot to put it back in.